builder/actions/setup_ci_from_files.py

```diff
@@ -0,0 +1,313 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+
+from builder.core.action import Action
+from builder.core.host import current_os, current_arch
+import os
+import pathlib
+import sys
+import json
+import tempfile
+import xml.etree.cElementTree as xml
```

Comment:
keep it simple
don't add complexity just because you can
we've never used XML for anything before, so don't start using it now

you can add comments in json like this:
`"_comment": "haha no one expected this field so I can put whatever I want here"`

Reply:
Will do. I will need to adjust the code a bit to skip fields starting with "_comment", but it is doable. I'll do such and remove XML parsing.